app/main.py

```diff
@@ -1,3 +1,47 @@
+from math import cos, sin, radians, degrees, acos, atan2
+
+
 class Vector:
-    # write your code here
-    pass
+    def __init__(self, x=0.0, y=0.0):
+        self.x = round(x, 2)
+        self.y = round(y, 2)
+
+    def __add__(self, other):
+        return Vector(round((self.x + other.x), 2),
+                      round((self.y + other.y), 2))
+
+    def __sub__(self, other):
+        return Vector(round((self.x - other.x), 2),
+                      round((self.y - other.y), 2))
+
+    def __mul__(self, other):
+        if isinstance(other, Vector):
+            return (self.x * other.x) + (self.y * other.y)
+        if isinstance(other, (int, float)):
+            return Vector(round((self.x * other), 2),
+                          round((self.y * other), 2))
+
+    @staticmethod
+    def create_vector_by_two_points(start_point: tuple, end_point: tuple):
+        return Vector(round((start_point[0] - end_point[0]) * -1, 2),
```

Comment:
You can simplify this expression
```
(start_point[0] - end_point[0]) * -1
```
```
end_point[0] - start_point[0] 
```
It would make your code more readable